# Data Storytelling

In [1]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

In [2]:
df = pd.read_csv('sephora_website_dataset.csv')

In [3]:
df.describe()

id       rating  number_of_reviews          love  \
count  9.168000e+03  9168.000000        9168.000000  9.168000e+03   
mean   1.962952e+06     3.990020         282.139180  1.627859e+04   
std    3.859714e+05     1.007707         890.642028  4.260651e+04   
min    5.000000e+01     0.000000           0.000000  0.000000e+00   
25%    1.819453e+06     4.000000          10.000000  1.600000e+03   
50%    2.072354e+06     4.000000          46.000000  4.800000e+03   
75%    2.230591e+06     4.500000         210.000000  1.380000e+04   
max    2.359685e+06     5.000000       19000.000000  1.300000e+06   

             price  value_price  online_only    exclusive  limited_edition  \
count  9168.000000   9168.00000  9168.000000  9168.000000      9168.000000   
mean     50.063237     51.82359     0.234839     0.264725         0.091841   
std      47.164989     49.45902     0.423921     0.441211         0.288817   
min       2.000000      2.00000     0.000000     0.000000         0.000000   
25%      24.000000     25.00000     0.000000     0.000000         0.000000   
50%      35.000000     35.00000     0.000000     0.000000         0.000000   
75%      59.000000     60.00000     0.000000     1.000000         0.000000   
max     549.000000    549.00000     1.000000     1.000000         1.000000   

       limited_time_offer  
count         9168.000000  
mean             0.000327  
std              0.018087  
min              0.000000  
25%              0.000000  
50%              0.000000  
75%              0.000000  
max              1.000000

Based on the data we obtained from the Sephora website, the total sales data on the Sephora website is 9,168 consumers. From all of this data, it can be seen that the average number of reviews given by consumers on the Sephora website is 282 reviews with the average rating obtained from the total product being 3.9 out of a rating scale of 5.0. In addition, the average price of products purchased by consumers is around 50 dollars.

In [4]:
df = pd.read_csv('Titanic.csv')

In [6]:
df.describe()

PassengerId    Survived      Pclass         Age       SibSp  \
count   891.000000  891.000000  891.000000  714.000000  891.000000   
mean    446.000000    0.383838    2.308642   29.699118    0.523008   
std     257.353842    0.486592    0.836071   14.526497    1.102743   
min       1.000000    0.000000    1.000000    0.420000    0.000000   
25%     223.500000    0.000000    2.000000   20.125000    0.000000   
50%     446.000000    0.000000    3.000000   28.000000    0.000000   
75%     668.500000    1.000000    3.000000   38.000000    1.000000   
max     891.000000    1.000000    3.000000   80.000000    8.000000   

            Parch        Fare  
count  891.000000  891.000000  
mean     0.381594   32.204208  
std      0.806057   49.693429  
min      0.000000    0.000000  
25%      0.000000    7.910400  
50%      0.000000   14.454200  
75%      0.000000   31.000000  
max      6.000000  512.329200

In [7]:
# Create a DataFrame showing the count of survived and not survived passengers by sex
survival_by_sex = df.groupby(["Survived", "Sex"]).size().unstack()

# Print the results
print("Number of Survived Passengers:")
print(survival_by_sex)

Number of Survived Passengers:
Sex       female  male
Survived              
0             81   468
1            233   109


This data shows the number of passengers who survived the Titanic disaster, sorted by gender. Overall, there were 109 male passengers who survived and 233 female passengers who survived. Looking at this data, it can be seen that there is a significant difference in survival rates between men and women. The number of female passengers who survived was far greater than male passengers.